great_expectations/expectations/util.py

```diff
@@ -64,282 +15,27 @@ def add_values_with_json_schema_from_list_in_params(
-    INSERT = "INSERT"
-    LOAD = "LOAD"
-    OVERWRITE = "OVERWRITE"
-    DIRECTORY = "DIRECTORY"
-    BY = "BY"
-    DISTRIBUTE = "DISTRIBUTE"
-    HAVING = "HAVING"
-    GROUP = "GROUP"
-    JOIN = "JOIN"
-    LIKE = "LIKE"
-    ORDER = "ORDER"
-    SORT = "SORT"
-    TABLESAMPLE = "TABLESAMPLE"
-    WHERE = "WHERE"
-    CASE = "CASE"
-    PIVOT = "PIVOT"
-    LATERAL = "LATERAL"
-    TRANSFORM = "TRANSFORM"
-    EXPLAIN = "EXPLAIN"
-    ADD = "ADD"
-    FILE = "FILE"
-    JAR = "JAR"
-    CACHE = "CACHE"
-    CLEAR = "CLEAR"
-    ANALYZE = "ANALYZE"
-    DESCRIBE = "DESCRIBE"
-    QUERY = "QUERY"
-    LIST = "LIST"
-    REFRESH = "REFRESH"
-    RESET = "RESET"
-    SET = "SET"
-    SHOW = "SHOW"
-    COLUMNS = "COLUMNS"
-    DATABASES = "DATABASES"
-    FUNCTIONS = "FUNCTIONS"
-    PARTITIONS = "PARTITIONS"
-    EXTENDED = "EXTENDED"
-    TABLES = "TABLES"
-    TBLPROPERTIES = "TBLPROPERTIES"
-    VIEWS = "VIEWS"
-    UNCACHE = "UNCACHE"
-    EXCEPT = "EXCEPT"
-    MINUS = "MINUS"
-    INTERSECT = "INTERSECT"
-    UNION = "UNION"
-    ALL = "ALL"
-    FROM = "FROM"
-    DISTINCT = "DISTINCT"
-    WITH = "WITH"
-    AS = "AS"
-    MAX = "MAX"
-    VALUES = "VALUES"
-    GROUPING = "GROUPING"
-    SETS = "SETS"
-    ROLLUP = "ROLLUP"
-    CUBE = "CUBE"
-    MIN = "MIN"
-    COUNT = "COUNT"
-    SUM = "SUM"
-    AVG = "AVG"
-    FILTER = "FILTER"
-    FIRST = "FIRST"
-    IGNORE = "IGNORE"
-    LAST = "LAST"
-    NATURAL = "NATURAL"
-    INNER = "INNER"
-    CROSS = "CROSS"
-    LEFT = "LEFT"
-    OUTER = "OUTER"
-    SEMI = "SEMI"
-    RIGHT = "RIGHT"
-    FULL = "FULL"
-    ANTI = "ANTI"
-    ON = "ON"
-    USING = "USING"
-    NOT = "NOT"
-    ESCAPE = "ESCAPE"
-    LIMIT = "LIMIT"
-    LENGTH = "LENGTH"
-    ASTERISK = "*"
-    ASC = "ASC"
-    DESC = "DESC"
-    NULLS = "NULLS"
-    ROWS = "ROWS"
-    OUT = "OUT"
-    OF = "OF"
-    RANGE = "RANGE"
-    EXPLODE = "EXPLODE"
-    EXPLODE_OUTER = "EXPLODE_OUTER"
-    INLINE = "INLINE"
-    INLINE_OUTER = "INLINE_OUTER"
-    POSEXPLODE = "POSEXPLODE"
-    POSEXPLODE_OUTER = "POSEXPLODE_OUTER"
-    STACK = "STACK"
-    JSON_TUPE = "JSON_TUPE"
-    PARSE_URL = "PARSE_URL"
-    RANK = "RANK"
-    DENSE_RANK = "DENSE_RANK"
-    PERCENT_RANK = "PERCENT_RANK"
-    NTILE = "NTILE"
-    ROW_NUMBER = "ROW_NUMBER"
-    RESPECT = "RESPECT"
-    CUME_DIST = "CUME_DIST"
-    LAG = "LAG"
-    LEAD = "LEAD"
-    NTH_VALUE = "NTH_VALUE"
-    FIRST_VALUE = "FIRST_VALUE"
-    LAST_VALUE = "LAST_VALUE"
-    PRECEDING = "PRECEDING"
-    BETWEEN = "BETWEEN"
-    AND = "AND"
-    CURRENT = "CURRENT"
-    FOLLOWING = "FOLLOWING"
-    UNBOUNDED = "UNBOUNDED"
-    WHEN = "WHEN"
-    THEN = "THEN"
-    ELSE = "ELSE"
-    FOR = "FOR"
+    # deprecated-v0.15.29
+    warnings.warn(
+        """The module great_expectations.expectations.util.py is deprecated as of v0.15.29 in v0.18. Please import \
+method add_values_with_json_schema_from_list_in_params from great_expectations.expectations.expectation.
+""",
```

Comment:
```suggestion
        (
            "The module great_expectations.expectations.util.py is deprecated as of v0.15.29 in v0.18."
            " Please import method add_values_with_json_schema_from_list_in_params from"
            " great_expectations.expectations.expectation."
        ),
```

An alternative way of dealing with long lines.

Reply:
PyCharm complains: `Unexpected type(s):(tuple[str, str, str], Type[DeprecationWarning])Possible type(s):(Warning, Any)(str, Optional[Type[Warning]])`